packages/sdks/snippets/react/src/components/CustomTabs.tsx

```diff
@@ -0,0 +1,75 @@
+/**
+ * https://www.builder.io/c/docs/custom-components-children
+ * src/components/customTabs.tsx
+ */
+
+import { useState } from 'react';
+import { Blocks, BuilderBlock } from '@builder.io/sdk-react';
+
+type Tab = {
+  tabName: string,
+  children: BuilderBlock[]
+}
+type TabProps = {
+  tabList: Tab[];
+  builderBlock: {
+    id: string | undefined;
+  };
+}
+
+const CustomTabs = (props: TabProps) => {
+  const [activeTab, setActiveTab] = useState(0);
+
+  const selectTab = (tab: number) => {
+    setActiveTab(tab);
+  };
+
+  return (
+    <div>
+      <h2>Custom Component with editable regions</h2>
+
+      <div>
+        <div>
+          {/* 
+            The tabList[] prop is an array that represents the tabs.
+            Each tab has a tabName, which is displayed as the button label,
+            and children, which are the content blocks rendered inside the tab.
```

Comment:
should we mention that its coming from `inputs` declared inside `component-info` and link them to some docs that they can refer?

Reply:
Done. Added a link